glide-core/src/protobuf/command_request.proto

```diff
@@ -38,225 +38,438 @@ message Routes {
+    ZRangeStore                    = 1322;
+    ZRank                          = 1323;
+    ZRem                           = 1324;
+    ZRemRangeByLex                 = 1325;
+    ZRemRangeByRank                = 1326;
+    ZRemRangeByScore               = 1327;
+    ZRevRange                      = 1328;
+    ZRevRangeByLex                 = 1329;
+    ZRevRangeByScore               = 1330;
+    ZRevRank                       = 1331;
+    ZScan                          = 1332;
+    ZScore                         = 1333;
+    ZUnion                         = 1334;
+    ZUnionStore                    = 1335;
+
+    //// Stream commands
+
+    XAck                           = 1401;
+    XAdd                           = 1402;
+    XAutoClaim                     = 1403;
+    XClaim                         = 1404;
+    XDel                           = 1405;
+    XGroupCreate                   = 1406;
+    XGroupCreateConsumer           = 1407;
+    XGroupDelConsumer              = 1408;
+    XGroupDestroy                  = 1409;
+    XGroupSetId                    = 1410;
+    XInfoConsumers                 = 1411;
+    XInfoGroups                    = 1412;
+    XInfoStream                    = 1413;
+    XLen                           = 1414;
+    XPending                       = 1415;
+    XRange                         = 1416;
+    XRead                          = 1417;
+    XReadGroup                     = 1418;
+    XRevRange                      = 1419;
+    XSetId                         = 1420;
+    XTrim                          = 1421;
+
+    //// String commands
+
+    Append                         = 1501;
+    Decr                           = 1502;
+    DecrBy                         = 1503;
+    Get                            = 1504;
+    GetDel                         = 1505;
+    GetEx                          = 1506;
+    GetRange                       = 1507;
+    GetSet                         = 1508; // deprecated in 6.2.0
+    Incr                           = 1509;
+    IncrBy                         = 1510;
+    IncrByFloat                    = 1511;
+    LCS                            = 1512;
+    MGet                           = 1513;
+    MSet                           = 1514;
+    MSetNX                         = 1515;
+    PSetEx                         = 1516; // deprecated in 2.6.12
+    Set                            = 1517;
+    SetEx                          = 1518; // deprecated in 2.6.12
+    SetNX                          = 1519; // deprecated in 2.6.12
+    SetRange                       = 1520;
+    Strlen                         = 1521;
+    Substr                         = 1522;
+
+    //// Transaction commands
+
+    Discard                        = 1601;
+    Exec                           = 1602;
+    Multi                          = 1603;
+    UnWatch                        = 1604;
+    Watch                          = 1605;
+
+    //// JSON commands
+
+    JsonArrAppend                  = 2001;
```

Comment:
I think we should split this PR into two:
1. clean up all the existing commands
2. add the JSON and VSS commands

Reply:
what do you mean in cleanup?

Reply:
refactor?  move the enums?  Cleanup?

Reply:
I just copy-pasted Very Big Enum into multiple places and fixed two broken refs (they were named incorrectly).